lib/board.rb

```diff
@@ -0,0 +1,42 @@
+require "pry"
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def reset!
+    @cells.clear
```

Comment:
No need to call clear as the contents of cells will be overwritten by the assignment.